resources/styles~300x300.css

```diff
@@ -129,4 +129,14 @@
 .circle_comfy {
     r: "35";
     fill: #10A30B;
-}
\ No newline at end of file
+}
+
+.tile-list-item {
+  height: 80;
+}
+
+.tile-list-item text {
+  height: 32;
+  font-family: System-Regular;
+  fill: "white";
+}
```

Comment:
I think you are missing the `style.css` for the ionic watch. The ionic watch in the emulator doesn't work